### DISCUSSION TITLE
Performance improvement suggestions

### DISCUSSION THREAD

**Discussion by madhur4127** [clangd/clangd#1239] [General]
Hey awesome clangd community,

I've been using clangd at work for years now and absolutely love it.

I specialise in writing low latency and performance sensitive software. I'd like to use my skills in giving back to this wonderful software.

I'll be great if someone can help me with few pointers or ideas.

PS: I think a performance issue tag would be better

**Comment by HighCommander4** [comment]
> PS: I think a performance issue tag would be better

Good idea! I created one, and added the label to performance-related issues I'm aware of:

https://github.com/clangd/clangd/labels/performance

Also added the label to discussions on this topic:

https://github.com/clangd/clangd/discussions?discussions_q=label%3Aperformance
